### PR TITLE
use python3.7 instead python3.5

### PR DESCRIPTION
**Pull request by fblackburn1**
not really sure what are impacts of this change, so be careful

### CODE REVIEW

**Review by fredericlepied — CHANGES_REQUESTED** [review]
We don't have Python 3.7 on Debian Stretch and Centos 7...

**Comment by fblackburn1** [comment]
> We don't have Python 3.7 on Debian Stretch and Centos 7...

Oh so sad. 
So to understand correctly, these config are only for the software factory VM?
There are no impact on the tests or whatever others jobs?
If so we can close the PR

File: nodepool/runC/debian-stretch-rootfs.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     {{ bwrap_command }} /bin/bash -lc "apt install -y --allow-unauthenticated gpgv"
     {{ bwrap_command }} /bin/bash -lc "rm -rf /var/lib/apt/lists/* /usr/share/dbus-1/system-services/org.freedesktop.systemd1.service.systemd"
     {{ bwrap_command }} /bin/bash -lc "apt-get update -y"
-    {{ bwrap_command }} /bin/bash -lc "apt-get install -y openssh-server rsync iproute git python-pip python3-pip traceroute libpq-dev python2.7 libpython2.7-dev python-pkg-resources python3.5 libpython3.5-dev python3-pkg-resources libldap2-dev libsasl2-dev libssl-dev"
+    {{ bwrap_command }} /bin/bash -lc "apt-get install -y openssh-server rsync iproute git python-pip python3-pip traceroute libpq-dev python2.7 libpython2.7-dev python-pkg-resources python3.7 libpython3.7-dev python3-pkg-resources libldap2-dev libsasl2-dev libssl-dev"
```
Comment (sileht) [inline nodepool/runC/debian-stretch-rootfs.yaml:21]: 3.7 does not exists in stretch

File: nodepool/elements/virt-customize/zuul.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@ COMMON="libldap2-dev libsasl2-dev libssl-dev linphone-nogtk python-dev lsof"
 
 if [[ $(cat /etc/debian_version) =~ ^9\. ]]; then
     # debian 9
-    PKGS="rsync iproute git virtualenv python-pip python3-pip traceroute libpq-dev python2.7 libpython2.7-dev python-pkg-resources python-yaml python3.5 libpython3.5-dev python3-pkg-resources sudo docker-ce docker-ce-cli containerd.io devscripts dirmngr build-essential $COMMON"
+    PKGS="rsync iproute git virtualenv python-pip python3-pip traceroute libpq-dev python2.7 libpython2.7-dev python-pkg-resources python-yaml python3.7 libpython3.7-dev python3-pkg-resources sudo docker-ce docker-ce-cli containerd.io devscripts dirmngr build-essential $COMMON"
```
Comment (sileht) [inline nodepool/elements/virt-customize/zuul.sh:29]: 3.7 does not exists in stretch

File: playbooks/config/check.yaml [hunk 3]
```diff
@@ -101,7 +101,7 @@
       block:
         - name: Check syntax errors in repoxplorer definition files
           command: >
-            env - /opt/rh/rh-python35/root/bin/repoxplorer-config-validate --config {{ config_root }}/defconf/defconf-repoxplorer.py
+            env - /opt/rh/rh-python37/root/bin/repoxplorer-config-validate --config {{ config_root }}/defconf/defconf-repoxplorer.py
```
Comment (sileht) [inline playbooks/config/check.yaml:104]: On software factory it's still 3.5

**Review by sileht — CHANGES_REQUESTED** [review]
I think none of these changes are needed.

**Comment by fblackburn1** [comment]
Ok thanks

**Pull request closed without merging**